### ISSUE TITLE
OflatBox should be OFlatBox

### ISSUE THREAD

**Issue by 1sra3l** [fltk-rs/fl2rust#14]
https://github.com/fltk-rs/fl2rust/blob/62e26384b84fb7a782fd4a30093cfb9c611bdec0/src/utils.rs#L51

Is this where the word is interpreted?
```rs
    let ret = match s.as_str() {
        "Vert fill" => "VerticalFill",
        "Horz fill" => "HorizontalFill",
        "Vert knob" => "VerticalNice",
        "Horz knob" => "HorizontalNice",
        // this would likely fix it
        "OflatBox" => "OFlatBox",
        _ => s.as_str(),
    };
```

**Comment by MoAlyousef** [comment]
It's interpreted here:
https://github.com/fltk-rs/fl2rust/blob/62e26384b84fb7a782fd4a30093cfb9c611bdec0/src/gen.rs#L261

**Comment by MoAlyousef** [comment]
FLUID creates an OFLAT_BOX which now generates an OFlatBox.

**Comment by 1sra3l** [comment]
@MoAlyousef one more issue here
the "down_box"
```rs
                        "down_box" => {
                            if let Some(p) = props.get(i.wrapping_sub(1)) {
                                if p != "label" {
                                    let temp =
                                        utils::global_to_pascal(utils::unbracket(&props[i + 1]));
                                    let temp = match temp.as_str() {
                                        "OflatBox" => "OFlatBox",
                                        "OshadowBox" => "OShadowBox",
                                        "RflatBox" => "RFlatBox",
                                        "RshadowBox" => "RShadowBox",
                                        _ => temp.as_str(),
                                    };
                                    imp += &format!(
                                        "\t{}.set_down_frame(FrameType::{});\n",
                                        &elem.ident, temp,
                                    );
                                }
                            } else {
                                let temp = utils::global_to_pascal(utils::unbracket(&props[i + 1]));
                                let temp = match temp.as_str() {
                                    "OflatBox" => "OFlatBox",
                                    "OshadowBox" => "OShadowBox",
                                    "RflatBox" => "RFlatBox",
                                    "RshadowBox" => "RShadowBox",
                                    _ => temp.as_str(),
                                };
                                imp += &format!(
                                    "\t{}.set_down_frame(FrameType::{});\n",
                                    &elem.ident, temp,
                                );
                            }
                        }
```

**Comment by MoAlyousef** [comment]
Oops. I’ll add it when I get home.

**Comment by MoAlyousef** [comment]
This should be now fixed in 0.4.15 :)
